bouncer/blacklist/views.py

```diff
@@ -1,6 +1,40 @@
 from django.shortcuts import render
 from django.http import HttpResponse
+from django.views import View
+from django.template import loader
+from blacklist import models
+from blacklist.brain import (
+    is_ip_blacklisted,
+    is_email_blacklisted,
+    is_email_host_blacklisted,
+)
 
-# Create your views here.
-def index(request):
-    return HttpResponse("Blacklist")
+
+class RequestView(View):
+    template = loader.get_template("blacklist/ip_request.html")
+
+    def get(self, request, *args, **kwargs):
+        context = {"request_list": []}
+        if "email" in self.request.GET:
+            context["request_list"].append("email")
+            context["request_list"].append(self.request.GET["email"])
+            if is_email_blacklisted(self.request.GET["email"]):
```

Comment:
Instead of an `if` `else` statement, you could use something like:

```python
result = "NO"
if is_email_blacklisted(self.request.GET["email"]):
    result = "YES"
results.append(
    ("email", self.request.GET["email"], result)
)
```